docs/en/02_Developer_Guides/01_Templates/05_Template_Inheritance.md

```diff
@@ -46,9 +46,10 @@ your entire project for the appropriate `.ss` files located in `template` direct
 It will each and prioritize templates in the following priority:
 
 1. mysite (or other name given to site folder)
-2. themes
-3. modules 
-4. framework.
+2. module-specific themes (e.g. themes/simple_blog)
```

Comment:
Is this definitely the right priority? I answered this a while back: http://stackoverflow.com/questions/24969798/blog-template-isnt-overridden - I'm not sure if I'm right but I'm guessing I would have tested at the time. I'm fairly certain that themes are higher priority then mysite.